.github/workflows/ci.yaml

```diff
@@ -17,26 +17,26 @@ jobs:
     name: Validate branch name
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Validate branch name
         run: packages/devops/scripts/ci/validateBranchName.sh
 
   validate_migration:
     name: Validate new migrations
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Validate new migration
         run: packages/devops/scripts/ci/validateNewMigrations.sh
 
   validate_tests:
     name: Validate tests
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
         with:
-          node-version: 20.11.1
+          node-version-file: '.nvmrc'
           cache: yarn
       - run: yarn set version 3.2.1
```

Comment:
Does the `actions/setup-node` installation of Node come with Corepack?[^footnote]

[^footnote]: I couldn’t find any docs in their repo, but from what I understand Corepack’s been included with Node for quite a few major versions now—except if installing with Homebrew

If so, we might be able to replace these with `corepack enable yarn` and have it read the Yarn version from `package.json` 🤔

(Disclaimer: I’m not saying this from experience; it just seems like something that _should_ work 😅)

Reply:
Seems to work! Thanks for the suggestion :)